neuralprophet/forecaster.py

```diff
@@ -1604,9 +1609,14 @@ def plot(
                 optional, overwrites the default plotting backend.
 
                 Options
-                * ``plotly``: Use plotly for plotting
+                * ``plotly-resampler``: Use the plotly backend for plotting in resample mode. This mode uses the
+                    plotly-resampler package to accelerate visualizing large data by resampling it. For some
+                    environments (colab, pycharm interpreter)plotly-resampler might not properly vizualise the figures.
+                    In this case, consider switching to 'plotly-auto'.
```

Comment:
To be more clear, I would formulate the last line:
`In this case, consider switching to 'plotly' or 'matplotlib'`